tests/unit/design-system/components/menu/Menu.spec.jsx

```diff
@@ -0,0 +1,195 @@
+import { render, fireEvent, screen } from '@testing-library/react'
+import Menu from '~scss-components/menu/Menu'
+
+const resourcesMenuItems = [
+  { title: 'Lesson' },
+  { title: 'Quiz' },
+  { title: 'Attachment' }
+]
+
+const resourcesMenuItemsWithCustomArgs = [
+  { title: 'Lesson', defaultOnItemClickArgs: { path: '/lesson' } }
+]
+
+const resourcesMenuItemsWithNestedItems = [
+  { title: 'Lesson', nestedMenuItems: [{ title: 'Art' }, { title: 'Math' }] }
+]
+
+const resourcesMenuItemsWithAdditionalInfo = [
+  { title: 'Lesson', additionalInfo: 'This is a lesson' }
+]
+
+const lessonIcon = <svg data-testid='lesson-icon' />
+
+const resourcesMenuItemsWithIcon = [{ title: 'Lesson', graphics: lessonIcon }]
+
+const noItemsCustomMessage = 'No items available.'
+
+describe('Menu Component', () => {
+  test('should render menu items', () => {
+    render(
+      <Menu
+        anchorEl={document.createElement('div')}
```

Comment:
Can you replace `document.createElement` with some real element, like you did with `lessonIcon`?

Reply:
Yeah, used data-testid as I did with lessonIcon, but since it must be rendered, also added logic to render it in beforeEach